cmd/kubelet/app/server.go

```diff
@@ -108,6 +110,7 @@ import (
 const (
 	// Kubelet component name
 	componentKubelet = "kubelet"
+	kubeletClientService = "kubelet-client"
```

Comment:
This should probably just be "kubelet" for everything in this PR (clients or servers)

Reply:
updated (removed, pass componentKubelet instead)